clamav/main.tf

```diff
@@ -26,6 +26,7 @@ resource "cloudfoundry_app" "clamav_api" {
   timeout      = 600
   strategy     = "rolling"
   docker_image = var.clamav_image
+  tags         = var.tags
```

Comment:
It looks like [tags does not exist on the cloudfoundry_app resource](https://registry.terraform.io/providers/cloudfoundry-community/cloudfoundry/latest/docs/resources/app) so please remove from `clamav`

Reply:
Fixed! Good catch